EpicLoot/EpicLoot.cs

```diff
@@ -385,6 +394,7 @@ public static ConfigFile GetConfigObject()
 
         public static void InitializeConfig()
         {
+            LoadJsonFile<IDictionary<string, object>>("translations.json", LoadTranslations);
```

Comment:
This is unfortunately wrong. We don't want to have server authoritative translations. Users should be allowed to use their native language regardless of the server language.

Reply:
If the server has mods that need EpicLoot translation additions, they should distribute the patches but not enforce it via ServerSync

Reply:
This has been fixed up.  translations.json are no longer Server Synced.